package.json

```diff
@@ -16,22 +16,28 @@
   },
   "scripts": {
     "start": "node index.js",
-    "lint": "eslint --cache ./cloud && eslint --cache index.js",
-    "lint-fix": "eslint --cache --fix ./cloud && eslint --cache --fix index.js",
-    "test": "jasmine"
+    "lint": "eslint --cache ./cloud && eslint --cache index.js && eslint --cache ./spec",
+    "lint-fix": "eslint --cache --fix ./cloud && eslint --cache --fix index.js && eslint --cache  --fix./spec",
+    "test": "jasmine",
+    "coverage": "nyc jasmine",
+    "prettier": "prettier --write '{cloud,spec}/{**/*,*}.js' 'index.js'",
+    "watch": "babel-watch index.js"
   },
   "engines": {
     "node": ">=4.3"
   },
   "devDependencies": {
-    "eslint": "^7.17.0",
-    "eslint-config-standard": "^16.0.2",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "babel-eslint": "^10.1.0",
-    "jasmine": "^3.6.3",
-    "mongodb-runner": "^4.8.0",
-    "bluebird": "^3.7.2"
+    "babel-eslint": "10.1.0",
+    "babel-watch": "7.4.0",
+    "bluebird": "3.7.2",
```

Comment:
I am wondering now why this repo suddenly depends on `bluebird`? Is this really necessary?